homeassistant/components/sfr_box/__init__.py

```diff
@@ -0,0 +1,53 @@
+"""SFR Box."""
+from __future__ import annotations
+
+from sfrbox_api.bridge import SFRBox
+from sfrbox_api.exceptions import SFRBoxError
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_HOST
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryNotReady
+from homeassistant.helpers import device_registry as dr
+from homeassistant.helpers.httpx_client import get_async_client
+
+from .const import DOMAIN, PLATFORMS
+from .coordinator import DslDataUpdateCoordinator
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up SFR box as config entry."""
+    box = SFRBox(ip=entry.data[CONF_HOST], client=get_async_client(hass))
+    try:
+        system_info = await box.system_get_info()
+    except SFRBoxError as err:
+        raise ConfigEntryNotReady(
+            f"Unable to connect to {entry.data[CONF_HOST]}"
+        ) from err
```

Comment:
The coordinator can already handle this on first request/refresh,thispart is  thus not  needed

Reply:
`system_get_info` is only called in setup - not via a coordinator
I will address this when I add "system" sensors.

Reply:
Addressed in #85039